akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -443,6 +467,14 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
       .toMat(new AeronSink(outboundChannel(outboundContext.remoteAddress), ordinaryStreamId, aeron, taskRunner, envelopePool))(Keep.right)
   }
 
+  def outboundLarge(outboundContext: OutboundContext): Sink[Send, Future[Done]] = {
+    // TODO not sure about this, just cargo culting here
```

Comment:
it's right, you could extract the first 3 lines to a outboundFlow if the duplication is disturbing